npm-debug.log

```diff
@@ -0,0 +1,37 @@
+0 info it worked if it ends with ok
```

Comment:
oops! we dont need to keep this file, we could add it to the gitignore. It's essentially the place you go if you want more detailed info about why something major blew up.